### PR TITLE
Addresses #2012 edit cerebellar granule cell and precursor

### PR DESCRIPTION
**Pull request by bvarner-ebi**
Addresses [#2012, part 1](https://github.com/obophenotype/cell-ontology/issues/2012#issuecomment-1623718859):

- edits to 'cerebellar granule cell'
- edits to 'granule cell precursor'

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -12936,11 +12937,13 @@ SubClassOf(obo:CL_0001030 obo:CL_0001060)
 
 # Class: obo:CL_0001031 (cerebellar granule cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") obo:IAO_0000115 obo:CL_0001031 "Granule cell that is part of the cerebellum.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:mah") Annotation(oboInOwl:hasDbXref "doi:10.12688/f1000research.15021.1") Annotation(oboInOwl:hasDbXref "doi:10.3389/fncir.2020.611841") obo:IAO_0000115 obo:CL_0001031 "An excitatory granule cell with a soma located in the granular layer of cerebellar cortex. A mature cerebellar granule cell has short dendrites with a characteristic claw-like appearance and a long axon that ascends to the molecular layer where it bifurcates (except in non-teleost fish, where it does not bifurcate) and extends mediolaterally to form parallel fibers.")
```
Comment (aleixpuigb) [inline src/ontology/cl-edit.owl:12940]: Consider adding 'granule cell of the cerebellum' as an exact synonym (doi:[10.3389/fncir.2020.611841](https://doi.org/10.3389/fncir.2020.611841))

**Review by aleixpuigb — COMMENTED** [review]
Please see inline comments.

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -12936,11 +12937,13 @@ SubClassOf(obo:CL_0001030 obo:CL_0001060)
...
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0001031 "BTO:0004278")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.3389/fncir.2020.611841") rdfs:comment obo:CL_0001031 "It has been noted in at least some mammalian species that granule cells are the most numerous cell type in the cerebellum and in the brain, with at least 99% of all cerebellar neurons being granule cells.")
```
[the remark this answers is not in the document]
Comment (bvarner-ebi) [reply src/ontology/cl-edit.owl:12942]: Thanks for suggestion, but I think the scope of the current comment is more appropriate for 'cerebellar granule cell'.

**Review by aleixpuigb — APPROVED** [review]

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
the synonym is under rdfs:comment

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**